src/components/scenes/Staking/StakeOptionsScene.tsx

```diff
@@ -38,9 +41,19 @@ export interface StakeOptionsParams {
 const StakeOptionsSceneComponent = (props: Props) => {
   const { navigation, route, wallet } = props
   const { currencyCode } = route.params
-  const stakePlugins = useSelector(state => state.staking.walletStakingMap[wallet.id].stakePlugins ?? [])
-  const stakePolicies = useSelector(state => state.staking.walletStakingMap[wallet.id].stakePolicies ?? [])
-  const stakePositionMap = useSelector(state => state.staking.walletStakingMap[wallet.id].stakePositionMap)
+  const [stakePlugins = []] = useAsyncValue(async () => await getStakePlugins(wallet.currencyInfo.pluginId))
+  const stakePolicies = stakePlugins.flatMap(stakePlugin =>
+    stakePlugin
+      .getPolicies({ pluginId })
+      .filter(
+        stakePolicy =>
+          !stakePolicy.deprecated &&
+          stakePolicy.stakeAssets.some(asset => asset.pluginId === wallet.currencyInfo.pluginId && asset.currencyCode === currencyCode)
+      )
+  )
+  const stakePositionMap = useSelector(state =>
+    state.staking.walletStakingMap[wallet.id] == null ? {} : state.staking.walletStakingMap[wallet.id].stakePositionMap
```

Comment:
This can be simplified as `state => state.staking.walletStakingMap[wallet.id]?.stakePositionMap ?? {}`